aries/aries_vcx_core/src/wallet/indy/restore_wallet_configs.rs

```diff
@@ -0,0 +1,53 @@
+use async_trait::async_trait;
+use serde::{Deserialize, Serialize};
+use vdrtools::{types::domain::wallet::default_key_derivation_method, Locator};
+
+use super::indy_utils::parse_key_derivation_method;
+use crate::{errors::error::VcxCoreResult, wallet::base_wallet::ImportWallet};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RestoreWalletConfigs {
+    pub wallet_name: String,
+    pub wallet_key: String,
+    pub exported_wallet_path: String,
+    pub backup_key: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub wallet_key_derivation: Option<String>,
+}
+
+#[async_trait]
+impl ImportWallet for RestoreWalletConfigs {
```

Comment:
Why are we calling the same concept different names (restore, import)?

Reply:
The struct renamed to `ImportWalletConfigs`